db/scripts/writeSeedFlag.js

```diff
@@ -0,0 +1,26 @@
+require('promise.prototype.finally').shim();
+const fs = require('fs');
+
+const getPool = require('../helpers/getPool');
+const floodsExists = require('./floodsExists');
+
+let masterClient;
+
+/**
+  Write a SEED_FLAG to a file.
+  This allows you to propagate that environment variable to the surrounding bash script scope.
+  Seeding will take place in a further step in the deploy.sh script.
+**/
+const writeSeedFlag = (client) => {
+  return floodsExists(client)
+  .then((result) => {
+    const shouldSeed = result ? "false" : "true";
+    fs.writeFileSync(`${__dirname}/../../deployment/seed_flag.tmp`, `export SEED_FLAG=${shouldSeed}`);
+  })
+}
+
+module.exports = writeSeedFlag
+
+if (require.main === module) {
+  commandLineRun(seed, "masterAdmin");
```

Comment:
Could you explain what this part does? I'm not sure what commandLineRun is doing here

Reply:
commandLineRun is just a wrapper for functions that are invoked straight from node, rather than being part of a larger chain of functions. It creates a safe wrapper for creating and destroying db pools.